go.mod

```diff
@@ -38,6 +38,7 @@ exclude sigs.k8s.io/cluster-api v0.3.14
 // TODO: Once the repo is public, remove this so we use a versioned module
 replace (
 	github.com/aws/eks-anywhere/release => ./release
+	github.com/containerd/containerd => github.com/containerd/containerd v1.5.9
```

Comment:
nit: it would be great to start adding comments every time we add a `replace` with at least the name of the direct  module dependency that is holding us back. Adding the reason (eg. a CVE) could also be useful, but that can also be retrieved from the commit/PR